### PR TITLE
Edits to file types article

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

File: docs/source/essentials/file-types.mdx [hunk 1]
```diff
@@ -70,25 +58,33 @@ type schema {
...
 
 type Query {
-  field: String
+  field: String @deprecated
   ...
 }
 ```
 
-This format is more compact and easier to read and modify. Another advantage is that the SDL format carries GraphQL directives, which the Json format doesn't (see [this issue](https://github.com/graphql/graphql-spec/issues/300))
+You can't use introspection to download an SDL schema directly from a GraphQL endpoint. However, Apollo Android can convert an introspection schema to SDL automatically if you specify the `.graphqls` extension in the `--schema` option:
```
Comment (martinbonnin) [inline docs/source/essentials/file-types.mdx:66]:
```suggestion
You can't use introspection to download a SDL schema directly from a GraphQL endpoint. However, Apollo Android can convert an introspection schema to SDL automatically if you specify the `.graphqls` extension in the `--schema` option:
```

**Review by martinbonnin — APPROVED** [review]
🤩 🤩 🤩

File: docs/source/essentials/file-types.mdx (hunk 1 above)
Comment (StephenBarlow) [reply docs/source/essentials/file-types.mdx:66]: Ah this is correct as-is, because "SDL" is pronounced "Ess Dee Ell", so it needs "an" instead of "a"

Comment (martinbonnin) [reply docs/source/essentials/file-types.mdx:66]: TIL, Thanks!

**Pull request merged**